app/serializers/tag_serializer.rb

```diff
@@ -1,3 +1,7 @@
 class TagSerializer < ApplicationSerializer
-  attributes :id, :name, :topic_count
+  attributes :id, :name, :topic_count, :staff
+
+  def staff
+    DiscourseTagging.staff_tag_names.include?(name)
```

Comment:
Did you check this isn't creating an N+1?

Reply:
yeah this looks pretty risky to me:

https://github.com/discourse/discourse/blob/74d2d4f658171bcab41ccd7257495e7e39fb45e4/lib/discourse_tagging.rb#L196-L202